.github/workflows/go-generate-check.yml

```diff
@@ -0,0 +1,50 @@
+name: Developer helper
+on: pull_request
+jobs:
+  go-generate-check:
+    name: Check if "go generate" has been run
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.x
+        id: go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.16
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+      - name: Install deps
+        id: install-deps
+        run: |
+          SOLC_BIN=solc-linux-amd64-v0.6.12+commit.27d51765
+          curl -OL https://binaries.soliditylang.org/linux-amd64/$SOLC_BIN
+          sudo mv $SOLC_BIN /usr/bin/solc
+          sudo chmod +x /usr/bin/solc
+
+        shell: bash
+      - name: Install devtools
+        id: install-devtools
+        run: make devtools
+      - name: Run go:generate
+        id: go-generate
+        run: |
+          # ignore generating trezor package based on this comment:
+          # https://github.com/ethereum/go-ethereum/blob/master/accounts/usbwallet/trezor/trezor.go#L21-L43
```

Comment:
@meowsbits please have a look at the reason I decide to skip the trezor generating code which also requires the protobuffer.

I assume that this code if/when needs to be generated we need to take extra manual care, for this reason I suggest to net even check it for differences